database/seeds/SquawkAssignmentTablesSeeder.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use App\Models\Squawks\Range;
+use App\Models\Squawks\SquawkGeneral;
+use App\Models\Squawks\SquawkRangeOwner;
+use App\Models\Squawks\SquawkUnit;
+use Illuminate\Database\Seeder;
+
+class SquawkAssignmentTablesSeeder extends Seeder
```

Comment:
Empty seeder

Reply:
Needed for now as the main DatabaseSeeder uses it